src/commands/project/pipeline.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
Why was this added back? Can't see any null assertions in the PR

Reply:
Hm, not sure. I removed it again. Something probably happened during rebase. Good eye!